Framework/API/inc/MantidAPI/SpectrumInfo.h

```diff
@@ -7,6 +7,7 @@
 #include <boost/shared_ptr.hpp>
 
 #include <vector>
+#include <set>
```

Comment:
Good catch, yes, this was leftover from something that was removed. It should be gone now.